### PR TITLE
Task/update lpa addresses

### PR DESCRIPTION
**Pull request by bryn-pins**
## Ticket Number

https://pins-ds.atlassian.net/browse/AAPD-

## Checklist

- [ ] Requires infrastructure changes
- [ ] I have updated the documentation accordingly
- [x] My commit history in this PR is linear
- [x] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `main` (please only [rebase](https://github.com/Planning-Inspectorate/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

File: packages/appeals-service-api/src/routes/v2/appeal-cases/_caseReference/lpa-questionnaire-submission/address/index.js [hunk 1]
```diff
@@ -1,10 +1,43 @@
...
+const { openApiValidatorMiddleware } = require('../../../../../../validators/validate-open-api');
+const lpaCanModifyCase = require('../lpa-can-modify-case');
 const asyncHandler = require('@pins/common/src/middleware/async-handler');
+
 const router = express.Router({ mergeParams: true });
 
-// Document upload routes
-router.post('/', asyncHandler(createSubmissionAddress));
-router.delete('/:addressId', asyncHandler(deleteSubmissionAddress));
+router.use(
```
Comment (Si3rr4wow) [inline packages/appeals-service-api/src/routes/v2/appeal-cases/_caseReference/lpa-questionnaire-submission/address/index.js:13]: DW about it in this PR but we've got this in a few places now. Maybe we make an authedRouter that has this middleware already on it?

**Review by Si3rr4wow — APPROVED** [review]
lgtm, just one q but I'm sure it's fine

File: packages/appeals-service-api/src/routes/v2/appeal-cases/_caseReference/lpa-questionnaire-submission/address/index.js (hunk 1 above)
Comment (bryn-pins) [reply packages/appeals-service-api/src/routes/v2/appeal-cases/_caseReference/lpa-questionnaire-submission/address/index.js:13]: that would make a lot of sense

File: packages/appeals-service-api/src/routes/v2/appellant-submissions/repo.js [hunk 2]
```diff
@@ -15,6 +17,41 @@ module.exports = class Repo {
...
+								}
+							}
+						}
+					}
+				}
+			});
+
+			if (!result.Appeal.Users.some((x) => x.userId.toLowerCase() === userId.toLowerCase())) {
+				throw ApiError.forbidden();
+			}
```
[the remark this answers is not in the document]
Comment (bryn-pins) [reply packages/appeals-service-api/src/routes/v2/appellant-submissions/repo.js:46]: I had a test with it with capitals after copying manually and it failed
I then made lower case but left this in as a safety

**Review by Si3rr4wow — APPROVED** [review]

**Pull request merged**